### ISSUE TITLE
Regex in filter requests

### ISSUE THREAD

**Issue by abbas1511** [httptoolkit/httptoolkit#82]
Regex while search. Cannot search for more or sets of filter string. With regex multiple combination can be filtered which helps is debugging smaller flows.

![image](https://user-images.githubusercontent.com/3536054/86460170-83cf1080-bd45-11ea-9c2a-ed321c8ecf30.png)

---

_Does this affect you too? Click below and add a :+1: to vote for this and help decide where HTTP Toolkit goes next, or [go vote on the other most popular ideas so far](https://github.com/httptoolkit/feedback/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc)._

**Comment by pimterry** [comment]
Good idea! Thanks, yes I'd love to do that. This probably fits within https://github.com/httptoolkit/feedback/issues/27, but I'll leave it as a separate issue here, since it might get implemented as a separate step on top of that.

**Comment by abbas1511** [comment]
Make sense, #27 & #82 would make search & inspection easy.

**Comment by pimterry** [comment]
@abbas1511 if you're still interested in this, do take a look at the new functionality implemented in #27. There isn't regex filtering, as that's more complicated from a UX point of view (we need to worry about valid regexes, for starters) but there are a lot of options for fine-tuned specific filtering, and I suspect that covers most regex cases.

For now I'm not intending to add regex filtering in the short term, but I am open to it if you or anybody else has some specific examples where the current filtering isn't quite sufficient, and I'd love to hear about that if so.

**Comment by pimterry** [comment]
After many years with no response, I think we can assume that the advanced filtering tools in #27 have covered this case sufficiently (docs here: https://httptoolkit.com/docs/reference/view-page/#filtering-intercepted-traffic) so I'll close this. If anybody has any concrete examples where regex would be useful and the existing tools aren't sufficient, do please share more info here.
